store/storetest/reaction_store.go

```diff
@@ -123,33 +116,29 @@ func testReactionDelete(t *testing.T, ss store.Store) {
 
 	_, err = ss.Reaction().Save(reaction)
 	require.Nil(t, err)
+
 	result, err := ss.Post().Get(reaction.PostId, false)
-	if err != nil {
-		t.Fatal(err)
-	}
+	require.Nil(t, err)
+
 	firstUpdateAt := result.Posts[post.Id].UpdateAt
 
-	if _, err = ss.Reaction().Delete(reaction); err != nil {
-		t.Fatal(err)
-	}
+	_, err = ss.Reaction().Delete(reaction)
+	require.Nil(t, err)
+
+	reactions, rErr := ss.Reaction().GetForPost(post.Id, false)
+	require.Nil(t, rErr)
+
+	assert.Equal(t, len(reactions), 0, "should've deleted reaction")
```

Comment:
you can use `assert.Len` here because it would be more "semantic" about what are you checking here.